test-workspace-proto/schema.proto

```
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package test;
+
+message User {
+    string name;
```

Buf (1.54.0)

field test.User.name: missing field tag number

(COMPILE)